Commands/InteractionServiceHandler.cs

```diff
@@ -79,7 +80,7 @@ public static async Task OnInteractionCreatedAsync(SocketInteraction interaction
 		}
 		catch (Exception e)
 		{
-			await Program.LogAsync("InteractionServiceManager", $"Error handling interaction. {e.Message}", LogSeverity.Error);
+			Log.Error($"Error handling interaction. {e.Message}");
```

Comment:
Just a small change for better log message consistency

```suggestion
			Log.Error($"Error handling interaction: {e.Message}");
```